components/raftstore/src/coprocessor/split_check/keys.rs

```diff
@@ -177,19 +186,22 @@ pub fn get_region_approximate_keys(db: &Arc<DB>, region: &Region) -> Result<u64>
 
     let start = keys::enc_start_key(region);
     let end = keys::enc_end_key(region);
-    let cf = box_try!(db.c().cf_handle(CF_WRITE));
-    let (_, keys) = get_range_entries_and_versions(db.c(), cf, &start, &end).unwrap_or_default();
+    let cf = box_try!(db.cf_handle(CF_WRITE));
```

Comment:
It's ok in this PR but what do you think to remove `cf_handle` from engine_traits? I think it's possible because `cf_handle` can be created by `db` and `cf_str`, so we can pass `db` and `cf_str` everywhere we need a `cf_handle`.

Reply:
Yes, I or somebody will remove cf_handle from engine_traits eventually.